connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SinkConnectorConfig.java

```diff
@@ -27,13 +28,20 @@
 
 public class SinkConnectorConfig extends ConnectorConfig {
 
-    public static final String TOPICS_CONFIG = "topics";
-    private static final String TOPICS_DOC = "";
+    public static final String TOPICS_CONFIG = SinkTask.TOPICS_CONFIG;
+    private static final String TOPICS_DOC = "List of topics to consume, separated by commas";
     public static final String TOPICS_DEFAULT = "";
     private static final String TOPICS_DISPLAY = "Topics";
 
+    private static final String TOPICS_REGEX_DOC = "Regular expression giving topics to consume." +
+        "<p/>Under the hood, the regex is compiled to a <code>java.util.regex.Pattern</code>. " +
+        String.format("<p/>Only one of %s or %s should be specified.", TOPICS_CONFIG, SinkTask.TOPICS_REGEX_CONFIG);
```

Comment:
Minor, but would using + operator be more consistent with the other lines that make up this literal?